probe-dictionary/explore.js

```diff
@@ -172,7 +172,7 @@ function showSearchOnlyFilters(show) {
     "#version-selection-element",
     "#optout-selection-element",
     "#text-search-element",
-    "#select_channel option[value=any]"
+    "#select_channel option[value=release]"
```

Comment:
Won't this hide the "any" choice from the search tab? It's a very helpful option, and should remain for search.

Reply:
@chutten , Though it doesn't hide `any' choice from the drop-down. But I have still set the `value` to `any` because it's still working without changing it.